src/main/kotlin/step4/racingcar/racingcar/GameResult.kt

```diff
@@ -0,0 +1,12 @@
+package step4.racingcar.racingcar
+
+data class GameResult(
+    val cars: List<Car>,
+    val round: Int
+) {
+    fun getWinner(): String {
+        val cars = this.cars
+        val maxCount = cars.maxOf { car -> car.getMovementCount() }
+        return cars.filter { car -> car.getMovementCount() == maxCount }.joinToString { it.racerName }
```

Comment:
```suggestion
        return cars.filter { car -> car.getMovementCount() == maxCount }
```

joinToString으로 하면, view가 domain에 침범하는 것처럼 보입니다. 차라리, winner를 배열로 보내주는 게 어떨까요? 🤔